contrib/podmanremoteimage/Containerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:latest AS builder
+WORKDIR /opt/app-root/src
+COPY . .
+RUN make podman-remote-static
+RUN GOOS=windows make podman-remote
+RUN GOOS=darwin make podman-remote
```

Comment:
What do you think of: Instead of compiling, curl the latest binaries from CI system using the branch-name specified as a build-arg?

Reply:
@cevich if we are adding this to CI for each PR then first we have to wait for other CI run to be succeed (to generate required images) as part of some step builds?

Reply:
Not the way the builds run today.  They're optional on `[CI:BUILD]` PRs (to support testing/development).  Otherwise they only run via a daily cron-job on `main`.